<launch>
  <env name="ROSCONSOLE_CONFIG_FILE" value="$(find reactive_grasping)/config/rosconsole_info_enabled.conf" />

  <rosparam command="load" file="$(find reactive_grasping)/config/comparison_dataset.yaml" />

  <!-- choose whether or not to enable the verbose mode -->
  <arg name="verbose_mode_glove" default="true"/>
  <arg name="verbose_mode_detection" default="true"/>
  <arg name="verbose_mode_motion" default="true"/>

 <!--  <include file="$(find glove_acquisition)/launch/glove_publisher.launch">
    <arg name="glove_pub_verbose_mode" value="$(arg verbose_mode_glove)"/>
  </include>
 -->
  <node pkg="reactive_grasping" type="detection_standalone" name="detection" output="screen" cwd="ROS_HOME">
    <param name="verbose_mode" type="bool" value="$(arg verbose_mode_detection)" />
    <param name="contact_threshold" type="double" value="120" />  <!-- previous set 130 -->
    <param name="false_positive_threshold" type="double" value="200" /> <!-- previous set 200 -->
    <param name="window_size" type="int" value="30" /> <!-- previous set 30 -->
    <param name="only_detection" type="bool" value="false" />
    <param name="calibration" type="bool" value="false" />
    <param name="glove_topic_name" type="string" value="/glove_data" />
    <param name="contact_direction_topic_name_" type="string" value="/which_finger" />
    <param name="topic_queue_length" type="int" value="0" />


  </node>

</launch>
